modules/renter.go

```diff
@@ -52,6 +52,7 @@ type FileInfo struct {
 	Filesize       uint64            `json:"filesize"`
 	Available      bool              `json:"available"`
 	Renewing       bool              `json:"renewing"`
+	Redundancy     float64           `json:"redundancy"`
 	UploadProgress float64           `json:"uploadprogress"`
```

Comment:
`UploadProgress` has higher granularity. With `Redundancy`, the step size is 1/TotalRedundancy, so about 5% with 24 pieces. With `UploadProgress`, it's technically 1 byte, but we only update after each chunk, so in reality it's 1/NumChunks (with redundancy).
So the issue is that with a sufficiently large file, its redundancy will appear stuck at 0 for a long time.